README.md

```diff
@@ -134,6 +134,12 @@ The first argument to the `type` function is the type or types (as an array) of
 
 You can return stuff from this callback too. Aargh will return this value to the caller.
 
+### .catch(errorTypes, callback)
```

Comment:
You know what, let's keep things simple:
1. Let's use `handle` instead of `catch`. I think that's clearer.
2. No need to add extra documentation, just replace `type()` with `handle`. We'll keep it in the code, but going forward, `handle` is the encouraged way.